skimage/measure/tests/test_regionprops.py

```diff
@@ -1369,6 +1368,7 @@ def test_column_dtypes_correct():
 
 
 def test_all_documented_items_in_col_dtypes():
+    numpydoc = pytest.importorskip("numpydoc")
     docstring = numpydoc.docscrape.FunctionDoc(regionprops)
```

Comment:
```suggestion
    numpydoc_docscrape = pytest.importorskip("numpydoc.docscrape")
    docstring = numpydoc_docscrape.FunctionDoc(regionprops)
```

in anticipation of the move to NumPy 2.0 (see https://github.com/scikit-image/scikit-image/pull/7288/commits/cf899e85448e8945af877c8eaff9fdc486e21a83 for context).